.github/workflows/operator-integration-tests-v2.yml

```diff
@@ -12,12 +12,50 @@ on:
         type: boolean
         default: false
         required: false
+      env:
+        type: choice
+        description: Choose the environment (i.e. which cluster to run the integration tests on)
+        required: true
+        options:
+          - CI
+          - dev
+
 
 concurrency:
   group: operator-e2e-tests
   cancel-in-progress: false
```

Comment:
Currently, concurrency is grouped by string `operator-e2e-tests`. This makes sure only one job is running at a time. Now that we have 2 envs, let's change group to consider `env` as well.

Reply:
Addressed.